mkat_fpga_tests/test_cbf.py

```diff
@@ -724,3 +725,69 @@ def test_vacc(self):
             Aqf.is_true(np.array_equal(expected_response, response),
                 'Check that the complexised response is equal'
                     ' to the expected response')
+
+    @aqf_vr('TP.C.1.40')
+    def test_product_switch(self):
+        """(TP.C.1.40) CBF Data Product Switching Time"""
+        test_name = '{}.{}'.format(strclass(self.__class__), self._testMethodName)
+        # Select dsim signal output, zero all sources, output scalings to 0.5
+        init_dsim_sources(self.dhost)
```

Comment:
Perhaps we should put init_dsim_source in setUp() and remove it from the beginning of all the other tests? Do in a separate PR.